src-tauri/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "phosphobot-desktop"
+version = "0.3.56"
+description = "The AI-ready robotics dev kit, with built-in remote control and VLA support"
+authors = ["phospho <contact@phospho.ai>"]
+edition = "2021"
+license = "MIT"
+repository = "https://github.com/phospho-app/phosphobot"
```

Comment:
does this version number also needs to be kept up to date ? Or can we put a dummy one, like in the dashboard? 
bumping version numbers accros files is very boring and error-prone.